resources/views/pages/login.twig

```diff
@@ -73,6 +74,11 @@
                                     {{ __('login.login') }}
                                 </button>
                             </div>
+                            {% else %}
+                            <div class="mb-3 text-center">
+                                <a href="{{ url(request.url(), {'password':1}) }}">…</a>
```

Comment:
That is a not-so-pretty workaround (as its only presumably used by admins when the OAuth is not working) and imho we should use the existing feature of "showing" stuff when something gets clicked which is used to show additional OAuth buttons in https://github.com/engelsystem/engelsystem/blob/main/resources/assets/js/forms.js#L273